evm.md

```diff
@@ -1065,13 +1037,15 @@ The `JUMP*` family of operations affect the current program counter.
 
     syntax UnStackOp ::= "JUMP"
  // ---------------------------
-    rule <k> JUMP DEST => #if OP ==K JUMPDEST #then #endBasicBlock #else #end EVMC_BAD_JUMP_DESTINATION #fi ... </k>
+    rule <k> JUMP DEST => #if DEST in DESTS #then #endBasicBlock #else #end EVMC_BAD_JUMP_DESTINATION #fi ... </k>
          <pc> _ => DEST </pc>
-         <program> ... DEST |-> OP ... </program>
+         <program> PGM </program>
+         <jumpDests> DESTS </jumpDests>
+      requires DEST <Int #sizeByteArray(PGM)
 
     rule <k> JUMP DEST => #end EVMC_BAD_JUMP_DESTINATION ... </k>
          <program> PGM </program>
-      requires notBool (DEST in_keys(PGM))
+      requires DEST >=Int #sizeByteArray(PGM)
```

Comment:
Can we just have one rule which says `DEST in DESTS` and the other which says `notBool DEST in DESTS`? No need to check the size of the program or anything assuming that we've always computed `#validJumpDestinations` ahead of time.